pkg/local_object_storage/pilorama/forest_test.go

```diff
@@ -202,6 +204,20 @@ func testForestTreeDrop(t *testing.T, s Forest) {
 
 	_, err = s.TreeGetByPath(cid, trees[1], AttributeFilename, []string{"path"}, true)
 	require.NoError(t, err)
+
+	for i := range trees {
+		_, err := s.TreeAddByPath(d, trees[i], AttributeFilename, []string{"path"},
```

Comment:
Can we add some nodes for multiple container IDs and also check that all other trees are still present?

Reply:
Thanks, added.